mmdet/models/dense_heads/corner_head.py

```diff
@@ -768,7 +774,7 @@ def _gather_feat(self, feat, ind, mask=None):
             feat (Tensor): Gathered feature.
         """
         dim = feat.size(2)
-        ind = ind.unsqueeze(2).expand(ind.size(0), ind.size(1), dim)
```

Comment:
May also add comments on the consequences if we use `expand` rather than `repeat` to remind the future refactoring to avoid expand.